bin/citadel

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2022 Citadel and contributors
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+set -euo pipefail
+
+CITADEL_ROOT="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/..)"
+CLI_NAME="$(basename $0)"
+CLI_VERSION="0.0.1"
+
+if [ -z ${1+x} ]; then
+  command=""
+else
+  command="$1"
+fi
+
+show_help() {
+  cat <<EOF
+${CLI_NAME}-cli v${CLI_VERSION}
+Automatically initialize and manage isolated Citadel instances.
+
+Usage: ${CLI_NAME} <command> [options]
+
+Flags:
+    -h, --help                         Show this help message
+    -v, --version                      Show version information for this CLI
+
+Commands:
+    debug                              View logs for troubleshooting
+EOF
+}
+
+# Debug Citadel
+if [[ "$command" = "debug" ]]; then
+  shift
+  sudo $CITADEL_ROOT/scripts/debug $@
+  exit
+fi
+
+# Update Citadel
+if [[ "$command" = "update" ]]; then
+  POSITIONAL_ARGS=()
+
+  branch=$(git rev-parse --abbrev-ref HEAD)
+  force=false
+
+  while [[ $# -gt 0 ]]; do
+    case $1 in
+    -b | --branch)
+      branch="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    --force)
+      force=true
+      shift # past argument
+      ;;
+    -* | --*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      POSITIONAL_ARGS+=("$1") # save positional arg
+      shift                   # past argument
+      ;;
+    esac
+  done
+
+  set -- "${POSITIONAL_ARGS[@]}" # restore positional parameters
+
+  if $force; then
+    sudo rm -f $CITADEL_ROOT/statuses/update-in-progress $CITADEL_ROOT/statuses/update-status.json
+  fi
+
+  sudo $CITADEL_ROOT/scripts/update/update --repo runcitadel/core#$branch
+
+  exit
+fi
+
+# Reboot the system
+if [[ "$command" = "reboot" ]]; then
+  sudo reboot
+  exit
+fi
+
+# Shutdown the system
+if [[ "$command" = "shutdown" ]]; then
+  sudo shutdown now
```

Comment:
It'd be a bit safer to run the stop script here (but with `|| true` so shutdown even happens if it fails)